.github/workflows/run-end-to-end.yml

```diff
@@ -115,6 +116,9 @@ jobs:
     - name: Run DEFAULT scenario
       if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"DEFAULT"')
       run: ./run.sh DEFAULT
+    - name: Run IPV6 scenario
+      if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"IPV6"') && !contains(fromJSON('["cpp", "dotnet", "java", "ruby", "ipv6"]'), inputs.library)
```

Comment:
Why ipv6 here?

`!contains(fromJSON('["cpp", "dotnet", "java", "ruby", "ipv6"]')`

Reply:
typoooo

Reply:
addressed in [3aae93a](https://github.com/DataDog/system-tests/pull/3624/commits/3aae93ab6692861a39a315dea348460a3119bb2b)